.github/ISSUE_TEMPLATE/content-review-request.md

```
@@ -0,0 +1,68 @@
+      description: Do you want the reviewer to reach out to you following the review
+        process?
+      options:
+        - Yes - I want to know the outcome
+        - No - I do not need to know the outcome
+  - type: input
+    id: input-3
+    attributes:
+      label: Contact Information (Optional)
+      description: This is required if you responded 'Yes' to the Post-Review
+        Follow-up question. Please provide and email address or Github username.
+      placeholder: "@Terry-Brooksjr  or API@yo-momma.net"
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)